storage/src/main/java/org/apache/kafka/storage/internals/log/SkimpyOffsetMap.java

```diff
@@ -109,7 +109,7 @@ public long get(ByteBuffer key) throws DigestException {
         hashInto(key, hash1);
         // search for the hash of this key by repeated probing until we find the hash we are looking for or we find an empty slot
         int attempt = 0;
-        int pos = 0;
+        int pos;
```

Comment:
maybe we can move `int pos;` into the loop?

Reply:
Makes sense. Moved the same in the next commit.
Thanks!